noir-projects/aztec-nr/aztec/src/state_vars/shared_mutable/scheduled_delay_change/test.nr

```diff
@@ -6,7 +6,7 @@ unconstrained fn assert_equal_after_conversion(original: ScheduledDelayChange<TE
     // We have to do explicit type annotations because Noir lacks turbofish support.
     // TODO: improve syntax once https://github.com/noir-lang/noir/issues/4710 is implemented.
     let converted: ScheduledDelayChange<TEST_INITIAL_DELAY> =
-        ScheduledDelayChange::deserialize((original).serialize());
+        ScheduledDelayChange::unpack(original.pack());
```

Comment:
The `test_serde` case should be renamed to e.g. `test_packable` or `test_pack_unpack`. This test likely also exists for other structs too.

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/11136/commits/a5abd16e00b700b885195d3c96d1016a0c05963c